lib/Db/Storage.php

```diff
@@ -12,10 +12,6 @@
 
 namespace OCA\Richdocuments\Db;
 
-/**
- * @method string loadRecentDocumentsForMimes()
- */
-
```

Comment:
This method is already a normal method in this class. phan complains about it being declared here in PHPdoc - that is not necessary. So I deleted this PHPdoc comment.